source/wp-content/themes/wporg-showcase-2022/src/style/style.scss

```diff
@@ -127,3 +127,21 @@ button[type="submit"] {
 	}
 
 }
+
+@media (max-width: 600px) {
```

Comment:
Why add this here rather than a `style.scss` in the block?

edit: I just re-read the description, and see that you'd rather not use CSS… can you use a columns block instead? that should switch to stacked at 600px. It just wouldn't keep the custom order.

Reply:
These styles are specific to the combination of the block and the search bar. The block itself is just the archive context sentence.

Reply:
I think we'll going to need this container functionality for other

> ```
> <!-- wp:columns {"verticalAlignment":"center","style":{"spacing":{"padding":{"top":"var:preset|spacing|40","bottom":"var:preset|spacing|40"}}}} -->
> <div class="wp-block-columns are-vertically-aligned-center" style="padding-top:var(--wp--preset--spacing--40);padding-bottom:var(--wp--preset--spacing--40)"><!-- wp:column {"verticalAlignment":"center","width":"50%"} -->
> <div class="wp-block-column is-vertically-aligned-center" style="flex-basis:50%"><!-- wp:wporg/archive-results-context /--></div>
> <!-- /wp:column -->
> 
> <!-- wp:column {"verticalAlignment":"center","width":"50%"} -->
> <div class="wp-block-column is-vertically-aligned-center" style="flex-basis:50%"><!-- wp:search {"showLabel":false,"placeholder":"<?php esc_attr_e( 'Search sites...', 'wporg' ); ?>","width":100,"widthUnit":"%","buttonText":"<?php esc_attr_e( 'Search', 'wporg' ); ?>","buttonPosition":"button-inside","buttonUseIcon":true,"className":"is-style-secondary-search-control"} /--></div>
> <!-- /wp:column --></div>
> <!-- /wp:columns -->
> ```

Additionally, the one problem I see with this approach is that the search bar will be variable width after passing the mobile breakpoint. We can control its width with column percentages but since they are percentage based, it will be a different sized component at 600px and 900px.  We also can't give it a width because it needs to be `100%` on mobile.

Reply:
You can set the column width in other units, like px or rem, and remove the width from the other block which will default it to the rest of the space. On smaller screens, it will still expand to the 100% width. Does that help?